src/Components/All.js

```diff
@@ -0,0 +1,135 @@
+import React, { useEffect, useState } from 'react';
+import { Grid } from '@mui/material';
+import { Paper } from '@mui/material';
+import { DeleteOutlined } from '@mui/icons-material';
+import ModeEditIcon from '@mui/icons-material/ModeEdit';
+import HighlightOffIcon from '@mui/icons-material/HighlightOff';
+import TextField from '@mui/material/TextField';
+import CheckIcon from '@mui/icons-material/Check';
+import Card from './Card';
+import './All.scss';
+import { callDeleteNoteApi, callEditNoteApi, callInitialDataApi } from '../Api';
+
+const All = () => {
+  const [notes, setNotes] = useState([]);
+  const [mouseEnter, setMouseEnter] = useState(false);
+  const [noteId, setNoteId] = useState();
+  const [isEditModeActive, setisEditModeActive] = useState(false);
+  const [activeEditNoteId, setIsActiveEditNoteId] = useState();
+  const [activeEditTitle, setIsActiveEditTitle] = useState();
+  const [activeEditDescription, setIsActiveEditDescription] = useState();
+
+  const onEditButtonClick = (title, description) => {
+    setIsActiveEditTitle(title);
+    setIsActiveEditDescription(description);
+  };
+
+  const titleEditor = (event) => {
+    setIsActiveEditTitle(event.target.value);
+  };
+
+  const descriptionEditor = (event) => {
+    setIsActiveEditDescription(event.target.value);
+  };
+
+  const editModeActivate = (id) => {
+    setIsActiveEditNoteId(id);
+    setisEditModeActive(true);
+  };
+
+  const editModeDeactivate = () => {
+    setIsActiveEditNoteId('');
+    setisEditModeActive(false);
+  };
+
+  const mouseEnterHandler = (id) => {
+    setNoteId(id);
+    setMouseEnter(true);
+  };
+  const mouseExitHandler = () => {
+    setNoteId('');
+    setMouseEnter(false);
+  };
+
+  const getInitialData = async () => {
+    const response = await callInitialDataApi();
+    var currentNotes = [...response.data.data];
+    setNotes([...currentNotes])
+  }
+
+  const deleteNoteApi = async () => {
+    const { status } = await callDeleteNoteApi(noteId);
+    if (status === 200) getInitialData();
+  };
+
+  const editNoteApi = async () => {
+    const { status } = await callEditNoteApi(activeEditNoteId, activeEditTitle, activeEditDescription);
+    if (status === 200) getInitialData(), editModeDeactivate(activeEditNoteId);
```

Comment:
Dont use commas like this. Sometimes it gets confusing. Also formatters can somehow behave weird and put these on 2 separate lines causing errors. Use braces for multiline ifs